src/sensors.c

```diff
@@ -131,6 +133,10 @@ static sensors_labeltypes_t known_features[] =
 	{ "2.0V", SENSOR_TYPE_VOLTAGE },
 	{ "12V", SENSOR_TYPE_VOLTAGE },
 	{ "power1", SENSOR_TYPE_POWER }
+	{ "curr1", SENSOR_TYPE_CURRENT }
```

Comment:
Older gcc versions seem to choke on this: `sensors.c:136: error: expected '}' before '{' token`. See the report from the jenkins build.